guides/common/modules/proc_acd_installation.adoc

```diff
@@ -9,7 +9,7 @@ Running ACD on Debian and Ubuntu is currently untested.
 endif::[]
 
 ifdef::foreman-el,katello[]
-This guide assumes you already have a working https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-foreman-el.html[Foreman] or https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-katello.html[Foreman and Katello] installation.
+This guide assumes you already have a working {BaseURL}Installing_Server_on_Red_Hat/index-foreman-el.html[Foreman] or {BaseURL}Installing_Server_on_Red_Hat/index-katello.html[Foreman and Katello] installation.
```

Comment:
Interesting. 
You've just shown us we have a missing entry for Katello Installation in our `attributes-base.adoc` file.
Out of scope for this but we must fix.